### PR TITLE
BottomSheet centered with maximum width.

### PR DESCRIPTION
**Pull request by etoledom**
Fixes: https://github.com/wordpress-mobile/gutenberg-mobile/issues/572

centered-bottom-sheet
simulator screen shot - ipad pro 9 7-inch - 2019-02-14 at 16 45 25

This PR updates the gutenberg ref to test https://github.com/WordPress/gutenberg/pull/13882

**Note:** Don't worry about the separators going out of the bottom sheet. That is resolved here: https://github.com/WordPress/gutenberg/pull/13855

**To test:**
Please refer to that PR for testing.

cc @iamthomasbishop

### CODE REVIEW

**Comment by iamthomasbishop** [comment]
This is looking great, @etoledom! Assuming the border-width issue and scrollable issues are being addressed separately, this one looks good to go.

File: src/block-management/block-picker.js [hunk 1]
```diff
@@ -4,18 +4,24 @@
...
+/**
+* Internal dependencies
+*/
 import styles from './block-picker.scss';
 import { name as unsupportedBlockName } from '../block-types/unsupported-block';
-// Gutenberg imports
-import { getBlockTypes } from '@wordpress/blocks';
 
 type PropsType = {
 	style?: StyleSheet,
```
Comment (pinarol) [inline src/block-management/block-picker.js]: looks like we don't need safeAreaBottomInset anymore

and we can also get rid of this at the bottom of the page:

BlockPicker.defaultProps = {
	safeAreaBottomInset: 0,
};

File: src/block-management/block-picker.js [hunk 2]
```diff
@@ -30,57 +36,46 @@ export default class BlockPicker extends Component<PropsType> {
 
 	render() {
 		const numberOfColumns = this.calculateNumberOfColumns();
-		const paddingBottom = this.paddingBottom();
+		const bottomPadding = this.props.addExtraBottomPadding && styles.contentBottomPadding;
```
Comment (pinarol) [inline src/block-management/block-picker.js]: we can add addExtraBottomPadding to PropsType

**Comment by etoledom** [comment]
Since this PR was so small, I took the opportunity to replace the BlockPicker Modal with the BottomSheet component, as a follow up of https://github.com/wordpress-mobile/gutenberg-mobile/pull/608

cc @iamthomasbishop 

picker

**Comment by etoledom** [comment]
Thank you!

**Comment by iamthomasbishop** [comment]
Looks great! One minor thing, but we can address it separately – the left/right sheet margins on the smaller portrait Android example are a bit narrow. Based on the screenshots I'm guessing we're using a static width for each block type which is fine for now, but we might want to adjust it to use percentages at some point to accommodate narrow screens.

**Comment by etoledom** [comment]
> I'm guessing we're using a static width for each block
Yes, width is static. If the screen is thin enough, the number of columns will be reduced to 2.
Of course, we can iterate and improve this :)

**Comment by iamthomasbishop** [comment]
Awesome, thanks for the confirmation!

**Pull request merged**